### PR TITLE
Add technical guide on contributing to Chimoney for Hacktoberfest 2024

### LINKED ISSUE #392: Write a Technical Article on How to Contribute to Chimoney Hacktoberfest 2024
Write a technical article that provides a step-by-step guide for developers to contribute to Chimoney's projects as part of Hacktoberfest 2024. This article will be targeted at both first-time open-source contributors and experienced developers. It should serve as an easy-to-follow guide for making meaningful contributions, and encourage the developer community to engage with Chimoney's repositories.

### **Content Outline:**

1. **Introduction to Hacktoberfest and Chimoney**:
    - What is Hacktoberfest?
    - Overview of Chimoney’s platform and API. 
    - Why open-source contributions matter to Chimoney.

2. **Step-by-Step Contribution Guide**:
    - Requirements for contributing.
    - How to set up the project locally:
        - Cloning the repository.
        - Running the project locally.

3. **Making Your First Contribution**:
    - How to choose an issue (using labels like `good first issue` or `help wanted`).
    - Being assigned an issue.
    - Contributing by writing code, or documentation.
    - How to create and submit a pull request (PR).
    - Best practices for clean and maintainable code.

4. **Opportunities for Contribution**:
    - Areas where Chimoney needs contributions (bug fixes, new features, documentation improvements).
    - Encouraging participants to create technical articles or guides around Chimoney's API or enhancing the developer experience.

5. **Contribution Guidelines**:
    - Overview of Chimoney’s contribution guidelines.
    - How to write meaningful commit messages.
    - Code of conduct.

6. **Benefits of Participating**:
    - What contributors gain from Hacktoberfest (learning, experience, Hacktoberfest swag).
    - How contributions impact Chimoney’s platform and mission.

7. **Next Steps After Contribution**:
    - Staying involved in the Chimoney community after Hacktoberfest.
    - Opportunities for continuous contributions to the Chimoney ecosystem.

### **Technical Requirements:**
- Written in markdown format with headers, bullet points, and links.
- Include relevant code snippets and screenshots where applicable.

### **Resources:**
- [Chimoney API Documentation](https://chimoney.readme.io/reference/introduction)
- [Hacktoberfest Official Website](https://hacktoberfest.com/)

### **How to Submit:**
Once completed, submit the article as a pull request in markdown format. Ensure it is well-structured and engaging for readers. Link the markdown file in the repository.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Danbaba1**
This pull request closes #392

### CODE REVIEW

**Comment by Danbaba1** [comment]
@phyleria Please review this pull request.

File: chimoney.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+# Contributing to Chimoney Projects: A Hacktoberfest 2024 Guide
+
+## 1. Introduction to Hacktoberfest and Chimoney
+
+### What is Hacktoberfest?
+
+Hacktoberfest is an annual event that celebrates open source software and encourages meaningful contributions to open source projects. It's sponsored by DigitalOcean and runs throughout the month of October. Participants who successfully complete the challenge by making a certain number of valid pull requests to open source repositories are eligible for exciting rewards and recognition.
```
Comment (phyleria) [inline chimoney.md:7]: Hi @Danbaba1, for this part `making a certain number` the number is 4 pull requests, please add that for clarity

File: chimoney.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+### How to Write Meaningful Commit Messages
+
+- Use the imperative mood (e.g., "Add feature" instead of "Added feature")
+- Keep the first line short (50 characters or less) and descriptive
+- Use the body of the commit message to explain the "why" behind the changes
+
+### Code of Conduct
+
+Chimoney follows a code of conduct that ensures a welcoming and inclusive environment for all contributors. Key points include:
```
Comment (phyleria) [inline chimoney.md:149]: In this section, please attach the code of conduct that is here: https://github.com/Chimoney/chimoney-community-projects/blob/main/CODE_OF_CONDUCT.md

File: chimoney.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+- Improve its products and services
+- Reach a wider audience of developers and businesses
+- Accelerate innovation in global payment solutions
+- Create a more robust and reliable platform for users worldwide
+
+## 7. Next Steps After Contribution
+
+### Staying Involved in the Chimoney Community
+
+- Join Chimoney's developer community on platforms like Discord or Slack
```
Comment (phyleria) [inline chimoney.md:177]: Hyperlink `Discord` using this link: https://discord.gg/TsyKnzT4qV

File: chimoney.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+- Participate in discussions and provide feedback on future developments
```
Comment (phyleria) [inline chimoney.md:178]: You can remove this

File: chimoney.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+- Attend Chimoney webinars or meetups to connect with other contributors
```
Comment (phyleria) [inline chimoney.md:179]: Please rephrase this `- Attend Chimoney webinars or meetups to connect with other contributors` to - Attend Chimoney's events [here](https://lu.ma/Chimoney) to connect with its global community and be updated with the community initiatives.

File: chimoney.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+
+### Opportunities for Continuous Contributions
+
+- Consider becoming a regular contributor or maintainer
```
Comment (phyleria) [inline chimoney.md:183]: Please remove `maintainer` here

File: chimoney.md [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+
+## 1. Introduction to Hacktoberfest and Chimoney
+
+### What is Hacktoberfest?
+
+Hacktoberfest is an annual event that celebrates open source software and encourages meaningful contributions to open source projects. It's sponsored by DigitalOcean and runs throughout the month of October. Participants who successfully complete the challenge by making a certain number of valid pull requests to open source repositories are eligible for exciting rewards and recognition.
+
+### Overview of Chimoney's Platform and API
+
+Chimoney is a fintech platform that provides seamless solutions for global payments, particularly focused on emerging markets. Chimoney also offers various programs like The Chimoney Ambassadorship Program and a recent initiative which is Chimoney for Startups, a program tailored to provide APIs and Payoutss for early-stage venture. The Chimoney API allows developers to integrate various payment and reward functionalities into their applications, including:
```
Comment (phyleria) [inline chimoney.md:11]: Please change this to 

> Chimoney is a global payouts and infrastructure provider offering an API that businesses, startups, and communities can integrate to handle bulk payouts, rewards, or disbursements in multiple currencies. Chimoney enables users to send money, gift cards, airtime, and other digital assets to recipients worldwide, making it easier for organizations of all sizes, especially those with distributed teams or diverse communities, to manage payments efficiently.

**Key features of Chimoney include:**

- **Bulk payouts**: Organizations can send payments to multiple recipients at once, with recipients able to redeem in their local currencies, saving time and effort.
- **Multi-currency support**: Chimoney supports a range of currencies, including USD, CAD, and NGN, for global transactions.
- **Payment requests**: Chimoney simplifies the process of requesting and receiving payments, making it easier for users to get paid.
- **API integration**: Developers can integrate Chimoney’s API into their systems, automating payment processes and scaling effortlessly.

**Review by phyleria — CHANGES_REQUESTED** [review]
.

**Review by phyleria — COMMENTED** [review]
Please submit this within the [Articles](https://github.com/Chimoney/chimoney-community-projects/tree/main/submissions/Articles) folder within submissions

**Comment by Danbaba1** [comment]
Okay @phyleria

**Comment by Danbaba1** [comment]
@phyleria I have made the changes you requested. Please review my pull request.

File: submissions/Articles/optimizing-chimoney-integration.md [hunk 8]
```diff
@@ -0,0 +1,315 @@
...
+- Follow the code of conduct and maintain a respectful, inclusive environment.
+- Keep pull requests focused on a single issue or feature.
+
+### How to Write Meaningful Commit Messages
+
+- Use the imperative mood (e.g., "Add feature" instead of "Added feature")
+- Keep the first line short (50 characters or less) and descriptive
+- Use the body of the commit message to explain the "why" behind the changes
+
+## Code of Conduct
```
Comment (phyleria) [inline submissions/Articles/optimizing-chimoney-integration.md:155]: Please hyperlink the code of conduct instead using the link I provided @Danbaba1

File: submissions/Articles/optimizing-chimoney-integration.md [hunk 9]
```diff
@@ -0,0 +1,315 @@
...
+- Improve its products and services
+- Reach a wider audience of developers and businesses
+- Accelerate innovation in global payment solutions
+- Create a more robust and reliable platform for users worldwide
+
+## 7. Next Steps After Contribution
+
+### Staying Involved in the Chimoney Community
+
+- Join Chimoney's developer community on platforms like [Discord](https://discord.gg/TsyKnzT4qV)
```
Comment (phyleria) [inline submissions/Articles/optimizing-chimoney-integration.md:306]: Rewrite this `- Join Chimoney's developer community on platforms like [Discord](https://discord.gg/TsyKnzT4qV)` to `- Join Chimoney's developer community on [Discord](https://discord.gg/TsyKnzT4qV)`

File: submissions/Articles/optimizing-chimoney-integration.md [hunk 10]
```diff
@@ -0,0 +1,315 @@
...
+- Attend Chimoney's events [here](https://lu.ma/Chimoney) to connect with its global community and be updated with the community initiatives.
```
Comment (phyleria) [inline submissions/Articles/optimizing-chimoney-integration.md:307]: Rewrite this `- Attend Chimoney's events [here](https://lu.ma/Chimoney) to connect with its global community and be updated with the community initiatives.` to `- Attend Chimoney's events [here](https://lu.ma/Chimoney) to connect with its global community and be updated on the community initiatives.`

**Review by phyleria — COMMENTED** [review]
Amazing, thank you for the changes! I have added just a few more suggestions and it will be good to go.

Also, can you rename the article from `optimizing-chimoney-integration.md` to `Contributing-to-Chimoney-Hacktoberfest24.md`

**Comment by Danbaba1** [comment]
@phyleria I have made the changes you requested. Please review my pull request.

**Comment by phyleria** [comment]
Great!
 
Please have the code of conduct section as:

> ## Code of Conduct
> Review Chimoney Community's Code of conduct [here](https://github.com/Chimoney/chimoney-community-projects/blob/main/CODE_OF_CONDUCT.md)

Since the code of conduct is now linked, you can remove the sections such as "Our Pledge," "Our Standards," etc. This will allow the reader to proceed directly to the "Benefits of Contributing" section after reviewing the Code of Conduct.

**Comment by Danbaba1** [comment]
Okay @phyleria

**Comment by Danbaba1** [comment]
@phyleria I have made the changes. Please review my pull request.

**Review by phyleria — APPROVED** [review]
LGTM 🎉

**Comment by Danbaba1** [comment]
Thank you so much @phyleria .

**Comment by Danbaba1** [comment]
@phyleria Are there any more issues to work on?

**Comment by Danbaba1** [comment]
@phyleria Am I getting a chimoney.io reward?

**Comment by phyleria** [comment]
Hi @Danbaba1,

Yes you will ✅

**Comment by Danbaba1** [comment]
@phyleria Thank you so much. I look forward to working on more open source projects with you. If there is any still available before hacktoberfest closes please let me know.

**Comment by phyleria** [comment]
Yes, I will let you know. Thank you very much for your contributions!

**Comment by Danbaba1** [comment]
@phyleria You are welcome.

**Pull request merged**